02-getting-account-data/index.js

```diff
@@ -0,0 +1,38 @@
+import { SDK } from 'NEWSDKNAME'
+
+// SDK exposes all you need like anchor-link does, users don't need to import from core directly, if they need something they do
+// import { Account } from 'NEWSDKNAME'
+
+const sdk = new SDK({ url: 'https://eos.greymass.com' }) // loaded with default plugins (something like if (browser) { plugins = [anchor-browser, atomichub] })
+
+async function main() {
+    const session = await sdk.login('myapp') // login also restores existing session unlike anchor-link, can have option for forcing re-login
+
+    // Loads account
+    const account = await session.accounts[0].get()
+    // Or you can load the account from the account name with either
+    // const account = await session.accounts.get('teamgreymass')
+    // or
+    // const account = await Account.get('eos', 'teamgreymass')
+
+    // Loads account balance
+    const balance = await account.getBalance('EOS')
+
+    // Load account resources
+    const { cpu, net, ram } = await account.getResources()
+
+    // Load account permissions
+    const permissions = await account.getPermissions()
```

Comment:
May also not need to `await` since this data is in the initial account object.

```js
const { permissions } = account

/*
{
  active: { ... permission },
  owner: { ... permission },
}
*/
```